pkgs/development/python-modules/glue-core/default.nix

```diff
@@ -0,0 +1,85 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+  astropy,
+  dill,
+  echo,
+  fast-histogram,
+  h5py,
+  ipython,
+  matplotlib,
+  mpl-scatter-density,
+  numpy,
+  openpyxl,
+  pandas,
+  pyqt-builder,
```

Comment:
It looks like this isn't required in `glue-core` anymore

Reply:
glue-core is the main library for glueviz application

Reply:
Yes, but `pyqt-builder` is not required to build `glue-core`.